### PR TITLE
Updated flake physics and suites, and change ccpp-physics branch to my ccpp-physics branch

### PR DESCRIPTION
**Pull request by YihuaWu-NOAA**
https://github.com/YihuaWu-NOAA/ccpp-physics

## Description

(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)
Provide a detailed description of what this PR does.  
What bug does it fix, or what feature does it add?  
Is a change of answers expected from this PR?  

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues should always be created before starting work on a PR branch!)
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested?  
What compilers / HPCs was it tested with?  
Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)  
Have the ufs-weather-model regression test been run? On what platform?  
- Will the code updates change regression test baseline? If yes, why? Please show the baseline directory below.
- Please commit the regression test log files in your ufs-weather-model branch

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
Those branches should have matching names (ideally)

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

File: ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noah.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noah.xml]: Is this file required? It's not related to flake. If it is, please fix line 3 in this SDF and all others in this PR. The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.

File: ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noah_flake.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_noah_flake" lib="ccppphys" ver="4">
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noah_flake.xml]: The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.

File: ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noahmp.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_noahmp" lib="ccppphys" ver="4">
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noahmp.xml]: Is this file required? It's not related to flake. If it is, please fix line 3 in this SDF and all others in this PR. The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.

File: ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noahmp_flake.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_noahmp_flake" lib="ccppphys" ver="4">
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noahmp_flake.xml]: The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.

File: ccpp/suites/suite_RRFS_parallel.xml [hunk 5]
```diff
@@ -0,0 +1,90 @@
+<suite name="RRFS_parallel" lib="ccppphys" ver="4">
```
Comment (climbfuji) [inline ccpp/suites/suite_RRFS_parallel.xml]: Please do not add this suite definition file, it is not related to flake and there are several RRFS-related suite definition files already.

**Review by climbfuji — CHANGES_REQUESTED** [review]
This PR contains one unrelated suite definition file that should be removed, all other SDFs use outdated syntax that needs to be fixed.

**Comment by YihuaWu-NOAA** [comment]
Hi, Dom,

Thanks.  I will delete these suites.




*Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
Systems Group
NOAA/NWS/NCEP/EMC
5830 University Research Court, Room: 2031
College Park, Maryland 20740, USA
Phone: (301)-683-3691
Fax:       (301)-683-3703

**Comment by climbfuji** [comment]
Replaced by https://github.com/NOAA-EMC/fv3atm/pull/287.
